src/db/visitor.test.ts

```diff
@@ -83,6 +83,7 @@ test('test that we get to the data nodes', async () => {
       fakeHash('none'),
       prevCommit.valueHash,
       prevCommit.indexes,
+      88,
```

Comment:
Please do not use this number for tests... it is associated with a symbol of hate.

Reply:
Whoops, forgot about that. For me it always means the number of miles per hour the delorean has to go to activate the flux capacitor. Will avoid in the future.